test/e2e/e2e_suite_test.go

```diff
@@ -39,8 +40,18 @@ func TestMain(m *testing.M) {
 	utilruntime.Must(err)
 
 	res := m.Run()
-	err = utils.PrintSummary(testSummaryOutputEnvVar)
-	utilruntime.Must(err)
+
+	path := os.Getenv(testSummaryOutputEnvVar)
+	if path == "" {
+		fmt.Printf("Note: E2E_SUMMARY_OUTPUT is unset; skipping summary generation")
+	} else {
+		err = utils.PrintSummary(path)
+		if err != nil {
+			// Fail the run if alerts are found
+			fmt.Printf("%s", err)
+			os.Exit(1)
```

Comment:
/hold
This actually causes an issue downstream. It seems that `PrintSummary` fails there. We need to come up with a solution that works for both upstream and downstream.
Also, did we want to use `%v` rather than `%s`?

Reply:
`PrintSummary` cannot return an error anymore, I disabled that to prevent it from failing the test. I can remove this section though if you want, but the idea was to have `PrintSummary` return an error for alerts once this is stable.

Reply:
So, #2144 merged, and you'll need to rebase onto that (it also had another fix).
If `PrintSummary` can no longer return an error, then it should have no return values.

Reply:
But, the function does not seem to return an error, but if it were to, it would still break because of the `os.Exti(1)`. That line should be removed; but for now, I'll approve.

Reply:
OK, the `E2E_SUMMARY_OUTPUT` variable won't be set downstream...